charts/vector/ci/vector-templating-values.yaml

```diff
@@ -16,8 +16,8 @@ customConfig:
       type: aws_s3
       inputs: [generator]
       bucket: logs-archive
-      key_prefix: |
-        {{ print "{{ kubernetes.pod_labels.app\\.kubernetes\\.io/client-id }}/%Y/%m/%d/{{ kubernetes.pod_name }}/" }}
+      key_prefix: >-
+        {{ print "{{kubernetes.pod_labels.app\\.kubernetes\\.io/client-id}}/%Y/%m/%d/{{kubernetes.pod_name}}/" }}
```

Comment:
@sam-mcbr this is how you are using Vector templates with helm, right?

Reply:
@spencergilbert Yeah